src/hooks/useFarmStats.js

```diff
@@ -53,7 +59,7 @@ export function useFarmStats(farmAddress) {
         }] = result.data.tokenGeysers
 
         // Calculate unlocked rewards per month.
-        const unlockRate = totalUnlockedRewards / 30;
+        const unlockRate = totalUnlockedRewards / DAYS_IN_MONTH;
```

Comment:
I was thinking about this... Is the contract assuming a 30 days fixed length? Or this is just for calculation purposes? If so, we might just wanna say "thirty days" because not necessarily every month has 30 days.

Reply:
It's just for calculation purposes - but you got a point, and we need to handle this in a better way.

Reply:
It's cool, just wanna make sure this is consistent in other parts of the UI as well. :)